src/pages/pt-br/reference/configuration-reference.md

```diff
@@ -257,6 +257,28 @@ Uma página markdown é considerada um rascunho se ela inclui `draft: true` em s
 }
 ```
 
+### markdown.mode
+
+<p>
+
+**Tipo:** `'md' | 'mdx'`<br>
+**Padrão:** `mdx`
+</p>
+
+Controle se o processamento de Markdown é feito utilizando MDX ou não.
+
+Processamento com MDX permite que você utilize JSX dentro de seus arquivos Markdown. Porém, podem haver momentos em que você não quer esse comportamento e prefira utilizar um processador "tradicional" de Markdown.
+
+MDX processing enables you to use JSX inside your Markdown files. However, there may be instances where you don't want this behavior, and would rather use a "vanilla" Markdown processor. This field allows you to control that behavior.
```

Comment:
```suggestion
```

Este texto em inglês deveria estar presente?

Reply:
No😅, thanks for catching this!